openmetadata-ui/src/main/resources/ui/src/utils/sanitize.utils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import DOMPurify from 'dompurify';
+
+export const getSanitizeContent = (html: string): string => {
+  return DOMPurify.sanitize(html);
```

Comment:
I feel we can directly utilise the lib function no need for additional util funtion

Reply:
@chirag-madlani, there are two reasons for doing that:
- To avoid repetitive imports of `DOMPurify`.
- To centralize all the sanitizing logic and configuration in one place.